.eslintrc.json

```diff
@@ -0,0 +1,105 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:import/errors",
+        "plugin:import/warnings"
+    ],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true,
+            "jsx": true
+        }
+    },
+    "plugins":  [
+        "react"
+    ],
+    "parser": "babel-eslint",
+    "rules":  {
+    "indent": [
+      "error",
+      2, {
+        "FunctionDeclaration": {"body": 1, "parameters": 2},
+        "FunctionExpression": {"body": 1, "parameters": 2},
+        "CallExpression": {"arguments": 1},
+        "MemberExpression": 1,
+        "SwitchCase": 1
+      }
+    ],
+    "linebreak-style": [
+      "error",
+      "unix"
+    ],
+    "operator-linebreak": [
+      "error",
+      "after"
+    ],
+    "quotes": [
+      "error",
+      "single",
+      {"avoidEscape": true}
```

Comment:
will this work with backticks?

Reply:
https://eslint.org/docs/rules/quotes
go through this, i think this would suffice.

Reply:
(y)